config/settings.yml

```diff
@@ -64,3 +64,4 @@ streaming_prefix:
   mpeg: 'mp4'
   m4a: 'mp4'
   mp3: 'mp3'
+image_viewer: Embed::Viewer::ImageX
```

Comment:
@mejackreed  could you please add a comment and indicate what the two options are?